pegasus/sites.v3/code.org/public/css/hoc2022-landing-page.scss

```diff
@@ -0,0 +1,299 @@
+@import "color.scss", "font.scss", "imports/_breakpoints.scss";
+
+html {
+  scroll-behavior: smooth;
+}
+
+#pageheader-wrapper #pageheader {
+  margin-bottom: 0;
+}
+
+// Allows for full-width banner and sections
+.container_responsive {
+  width: unset;
+  padding: unset;
+}
+
+header {
+  text-align: center;
+  padding: 5em 1em;
+  color: $white;
+  background: url("/images/hoc2022-landing-page-banner-bg.png") no-repeat center center;
+  background-size: cover;
+  position: relative;
+  overflow: hidden;
+  text-shadow: 0px 0px 8px #5d1582;
+
+  @media screen and (max-width: $width-md) {
+    padding: 3.5em 1em;
+  }
+
+  @media screen and (max-width: $width-sm) {
+    background: url("/images/hoc2022-landing-page-banner-bg-mobile.png") no-repeat bottom center;
+    background-size: cover;
+  }
+
+  .wrapper {
+    z-index: 1;
+    position: relative;
+    margin: unset;
+
+    @media screen and (max-width: $width-md) {
+      font-size: 88%;
+    }
+
+    a {
+      text-decoration: underline;
+
+      &:hover {
+        text-decoration: none;
+      }
+    }
+  }
+
+  img {
+    z-index: 0;
+    height: 100%;
+    position: absolute;
+    bottom: 0;
+
+    @media screen and (max-width: $width-sm) {
+      display: none;
+    }
+
+    &.left {
+      left: 0;
+
+      @media screen and (max-width: $width-lg) {
+        left: -62px;
+      }
+    }
+
+    &.right {
+      right: 0;
+
+      @media screen and (max-width: $width-lg) {
+        right: -48px;
+      }
+    }
+  }
+
+  h1, h2 {
+    color: $white;
+  }
+
+  h1 {
+    font-size: 4em;
+    margin: 0.5em auto;
+
+    @media screen and (max-width: $width-lg) {
+      font-size: 3.5em;
+    }
+
+    @media screen and (max-width: $width-sm) {
+      font-size: 4.5em !important; // overrides brstrap.css
+      margin: 0.25em auto -0.25em !important; // overrides brstrap.css
+    }
+  }
+
+  h2 {
+    font-family: $gotham-regular;
+  }
+
+  span {
+    font-size: 1.75em;
+  }
+
+  nav ul {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 380px;
+    margin: 0 auto;
+
+    @media screen and (max-width: $width-sm) {
+      justify-content: space-evenly;
+    }
+
+    li {
+      list-style: none;
+      display: inline-flex;
+      font-size: 1.5em;
+
+      &:before {
+        display: none;
+      }
+
+      i {
+        margin-right: 1em;
+      }
+
+      a {
+        margin-bottom: 0;
+        line-height: 1;
+        color: $white;
+        font-family: $gotham-bold;
+
+        &:visited,
+        &:link {
+          color: $white;
+        }
+      }
+    }
+  }
+}
+
+main {
+
+  // NASA skinny banner
+  aside {
```

Comment:
Am I reading it right that this is hidden? How did you test this code?

I'm in favor of not having "dead code" (i.e. code that isn't used) in our code base. Is it clear that we'll be needing this again soon?

Reply:
Sounds good!